.eslintrc.json

```diff
@@ -0,0 +1,60 @@
+{
+    "env": {
+        "browser": true,
+        "commonjs": true,
+        "es6": true
+    },
+    "extends": [
+        "airbnb",
+        "eslint:recommended",
```

Comment:
I suggest not to include `eslint:recommended` which seems to overwrite lots of rules we used to adopting.

Reply:
Additionally, inappropriate `indentation` was found in this file. 
Not `4 spaces`, `2 spaces` instead.

```json
{
  "env": {
    ...
  }
}
```